src/main/java/org/craftercms/engine/security/ConfigAwareAuthenticationHeadersLoginProcessor.java

```diff
@@ -80,30 +80,27 @@ public void setGroupsHeaderName(final String groupsHeaderName) {
     @Override
     public void processRequest(final RequestContext context, final RequestSecurityProcessorChain processorChain)
         throws Exception {
+        HttpServletRequest request = context.getRequest();
+        Authentication auth = SecurityUtils.getAuthentication(request);
+
+        logger.debug("Checking headers from Mellon");
```

Comment:
Can we not call it `Mellon`? There might be cases where something else is used instead of `Mellon`.